src/app/navbar/navbar.component.ts

```diff
@@ -58,13 +70,33 @@ export class NavbarComponent extends Logout implements OnInit {
       .subscribe(() => {
         this.isExtended = toExtendSite(this.router.url);
       });
+    this.isAdminOrCurator$ = this.userQuery.isAdminOrCurator$;
   }
 
   ngOnInit() {
     this.userQuery.user$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((user) => {
       this.user = user;
+      this.requestsService.updateMyMemberships();
+      this.requestsService.updateCuratorOrganizations();
     });
     this.userQuery.extendedUserData$.pipe(takeUntil(this.ngUnsubscribe)).subscribe((extendedUser) => (this.extendedUser = extendedUser));
+    this.myOrganizationInvites$ = this.requestsQuery.myOrganizationInvites$;
+    this.myRejectedOrganizationRequests$ = this.requestsQuery.myRejectedOrganizationRequests$;
+    this.allPendingOrganizations$ = this.requestsQuery.allPendingOrganizations$;
+    combineLatest([this.myOrganizationInvites$, this.myRejectedOrganizationRequests$, this.allPendingOrganizations$]).subscribe(
+      ([invites, rejections, pendingOrgs]: [Array<OrganizationUser>, Array<OrganizationUser>, Array<Organization>]) => {
+        this.isAdminOrCurator$.subscribe((isAdminorCurator: boolean) => {
```

Comment:
> 1. Also, I think you should use `pipe` and `takeUntil`, although I'm not 100% sure if the `takeUntil` is necessary
> 2. You could add `this.isAdminOrCurator$` to the `combineLatest`
> 3. You could also use `map`:
> 
> ```
> public notifCount$: Observable<number>
> ...
> this.notifCount$ = combineLatest([this.myOrganizationInvites$, this.myRejectedOrganizationRequests$, this.allPendingOrganizations$, this.isAdminOrCurator$])
>         .pipe(
>             takeUntil(this.ngUnsubscribe),
>             map(
>                 ([invites, rejections, pendingOrgs, isAdminOrCurator]: [Array<OrganizationUser>, Array<OrganizationUser>, Array<Organization>, boolean]) => {
>                   if (isAdminOrCurator) {
>                     return invites?.length + rejections?.length + pendingOrgs?.length;
>                   } else {
>                     return invites?.length + rejections?.length;
>                   }
>                 })
>         )
> ```
> 
> I feel strongly about 1 (unless proven otherwise), less strongly about 2, and even less strongly about 3 (the last one is personal preference and the way I would write it :) ).

Regarding item 3, you could make the code shorter and eliminate some duplication by rewriting the `if` as:
```
return invites?.length + rejections?.length + isAdminOrCurator ? pendingOrgs?.length : 0;
```
Might need (or want to add) parentheses, I always forget the precedence of the ternary operator...